packages/core/data/oot/world_mq/fire_temple_mq.yml

```diff
@@ -25,7 +25,7 @@
 "Fire Temple Vanilla Hammer Loop":
   dungeon: Fire
   locations:
-    "MQ Fire Temple Hammer Chest": "has_weapon && soul_keese && soul_enemy(SOUL_ENEMY_STALFOS) && soul_iron_knuckle && soul_enemy(SOUL_ENEMY_FLARE_DANCER) && (has_bombs || can_hammer || can_hookshot)"
+    "MQ Fire Temple Hammer Chest": "is_adult && has_weapon && soul_keese && soul_enemy(SOUL_ENEMY_STALFOS) && soul_iron_knuckle && soul_enemy(SOUL_ENEMY_FLARE_DANCER) && (has_bombs || can_hammer || can_hookshot)"
```

Comment:
instead of just checking for adult, can do ageless hookshot as well. i did that for vanilla fire too. there is also climb anywhere.

Reply:
So replace:
`is_adult`

With:
`(is_adult || can_hookshot || climb_anywhere)`

Makes sense. I'm pretty sure child hookshot, using equip swap, is how I rescued a seed that bottlenecked at this check. I've not played with climb anywhere, so I can't tell you it definitely will work; if you'd require verification.

I do note that climb_anywhere isn't anywhere in the fire_mq file. Is there concern that checking for `climb_anywhere` might give a future editor a false impression that someone had already completely accounted for climb anywhere within this dungeon?

(Apologies for the tardiness in my reply, I've been away from home over the Christmas period)

Reply:
considering im the one that did all the climb/hook anywhere and ageless logic, that shouldn't be an issue. i know i didnt do any mq for those.